dhis-2/dhis-support/dhis-support-commons/src/main/java/org/hisp/dhis/commons/util/SqlHelper.java

```diff
@@ -49,6 +49,8 @@
 
     private boolean andInvoked = false;
```

Comment:
Unused?

Reply:
Removed any changes to this class, since we ended up not using them :)